src/renderer/GPURenderer/Mobile/index.js

```diff
@@ -272,21 +271,18 @@ export default class MobileGPURenderer extends BaseRenderer {
 
     return this;
   }
-}
 
-MobileGPURenderer.getTextureID = (renderer, texture, debug) => {
-  if (texture.textureIndex === undefined) {
-    let atlas = MobileGPURenderer.textureAtlas;
+  getTextureID(texture, debug) {
+    if (texture.textureIndex === undefined) {
+      let atlas = this.textureAtlas;
```

Comment:
I think this can be simplified to 

```
const { textureAtlas = new TextureAtlas(this, debug) } = this;

textureAtlas.addTexture(texture);
```

I will try it out and see

Reply:
Ah nope, still need to set it

```
      const { textureAtlas } = this;

      if (!textureAtlas) {
        this.textureAtlas = new TextureAtlas(this, debug);
      }

      this.textureAtlas.addTexture(texture);
```

Works but is much the same, I'll just leave it